data/thebarshablog/events/2023-02-21-Decoding-CFPs-Twitter-Space.json

```diff
@@ -0,0 +1,12 @@
+{
+  "isVirtual": true,
+  "isInPerson": false,
+  "color": "purple",
+  "name": "Decoding CFPs (Call for Proposals)",
+  "description": "Twitter Space",
+  "date": {
+    "start": "2023-02-21",
```

Comment:
Shouldn't you add the starting hour and finishing hour?

Reply:
Yes I think this is important 👍

Reply:
@eddiejaoude & @Panquesito7 I made necessary changes.